README.md

```diff
@@ -95,9 +95,12 @@ sdr update druid:bb408qn5061 --url https://sdr-api-server:3000 --license "https:
 # Change access controls
 sdr update druid:bb408qn5061 --url https://sdr-api-server:3000 --view "location-based" --download "none" --location "music" --cdl false
 
-# Change Cocina wholesale from a file (note that you can use this flag with the
-# others above, and the flags above will replace what's supplied in the cocina file)
+# Change Cocina wholesale, either:
+# 1. From a file; or
```

Comment:
This reads strange to me, is the `; or` referring to #2?

Reply:
Yes 😄 But I agree it's janky so I'll clean it up.

Reply:
done!